TVCROLLDAP.s

```diff
@@ -136,7 +136,7 @@
 #	PARABOLA WHICH PASSES THROUGH THE OPERATING POINT.  FOR REGION 3
 #	DESIRED OGARATE IS SIMPLY +-MAXLIM.  FOR REGIONS 1 OR 6 THE SOLUTION
 #	TO A QUADRATIC IS REQUIRED (THE PENETRATION IS ALONG THE STRAIGHT
-#	LINE OR MINLIM BOUNDRY SWITCH LINES).  AN APPROXIMATION IS MADE
+#	LINE OR MINLIM BOUNDARY SWITCH LINES).  AN APPROXIMATION IS MADE
```

Comment:
`BOUNDRY` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0987.jpg)